### PR TITLE
fixedname

### LINKED ISSUE #1: workflow has wrong name!
**Description:**
workflow has wrong name

**Action version:**
Specify the action version

**Platform:**
- [ ] Ubuntu
- [ ] macOS
- [ ] Windows

**Runner type:**
- [ ] Hosted
- [ ] Self-hosted

**Tools version:**

**Repro steps:**  
A description with steps to reproduce the issue. If you have a public example or repo to share, please provide the link.

**Expected behavior:**
A description of what you expected to happen.

**Actual behavior:**
A description of what is actually happening.

### PR DESCRIPTION
**Pull request by gilesbradshaw**
**Description:**
Describe your changes.

**Related issue:**

fixes #1 

**Check list:**
- [ ] Mark if documentation changes are required.
- [ ] Mark if tests were added or updated to cover the changes.

### CODE REVIEW

**Review by gilesb2 — APPROVED** [review]
this is good

**Review by gilesb2 — APPROVED** [review]
i approve this

**Comment by gilesbradshaw** [comment]
just thinking - is echo a good name?

File: .github/workflows/echo-push.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Release new action version
+name: echo
```
Comment (gilesbradshaw) [inline .github/workflows/echo-push.yml:1]: is this a code conversation?

Comment (gilesb2) [reply .github/workflows/echo-push.yml:1]: yes!

**Pull request merged**